crates/nu-protocol/src/shell_error.rs

```diff
@@ -852,8 +861,8 @@ mod tests {
             (
                 vec!["OS", "PWD", "PWDPWDPWDPWD"],
                 vec![
-                    ("pwd", Some("PWD"), "Exact case insensitive match yields a match"),
-                    ("pwdpwdpwdpwd", Some("PWDPWDPWDPWD"), "Exact case insensitive match yields a match"),
+                    ("pwd", None, "The same item doesn't means match"),
+                    ("pwdpwdpwdpwd", None, "The same item doesn't means match"),
```

Comment:
I am not sure what you mean here.

Reply:
Sorry for my poor english, but I've change the case, it should yield a match